.github/CODEOWNERS

```diff
@@ -81,21 +81,22 @@
 /moveit_ros/move_group/	                                    @rhaschke @IanTheEngineer
 /moveit_ros/visualization/	                            @rhaschke @jonbinney @christian-rauch
 
-/moveit_ros/planning/collision_plugin_loader/               @rhaschke
-/moveit_ros/planning/constraint_sampler_manager_loader/     @henningkayser
+/moveit_ros/planning/collision_plugin_loader/               @j-petit
+/moveit_ros/planning/constraint_sampler_manager_loader/     @nbbrooks
```

Comment:
@nbbrooks to get you more involved, i've set you as reviewer of various plugin loaders ;-)